package.json

```diff
@@ -80,15 +80,14 @@
         "@commitlint/cli": "^16.2.3",
         "@commitlint/config-conventional": "^16.2.1",
         "@nguniversal/builders": "^12.1.3",
-        "@nrwl/angular": "13.8.6",
-        "@nrwl/cli": "13.8.6",
-        "@nrwl/eslint-plugin-nx": "13.8.6",
-        "@nrwl/jest": "13.8.6",
-        "@nrwl/js": "13.8.6",
-        "@nrwl/node": "13.8.6",
-        "@nrwl/tao": "13.8.6",
-        "@nrwl/workspace": "13.8.6",
-        "@nx-cloud/runners": "1.1.0",
+        "@nrwl/angular": "13.1.4",
+        "@nrwl/cli": "13.1.4",
+        "@nrwl/eslint-plugin-nx": "13.1.4",
+        "@nrwl/jest": "13.1.4",
+        "@nrwl/js": "13.3.0",
```

Comment:
The minimum available version of this package:
https://www.npmjs.com/package/@nrwl/js